commands/license_info

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env bash
+help() {
+  cat <<EOF
+
+----------------------------------------------------------------------------------------------------------
+HELP MESSAGE:
+
+    Usage: c8y oee $(basename "$0") [--output <json|table|csv|csvheader>] [--filePath <directory-to-store-csv-file>] [--select <keys-list>] [--help]
+
+    Arguments:
+      -o, --output        print-format      The printing format for data (json, table, csv file w/o header) (default: table)
+      -f, --filePath      directory         Directory to store csv file (default: current script directory c8y-oee/commands)
+      -s, --select        keys-list         Data to show (default: show all data) (run the script to see the list of key to use)
+      -h, --help                            Show this help message
+
+    Example usage:
+
+    Check license info and print in json format:
+        c8y oee license_info -o json
+    Check license info and print csv file with select data license status and license expiration:
+        c8y oee license_info -o csvheader -f "C:/path/to/storage/directory" -s 'licenseStatus, licenseExpiration'
+----------------------------------------------------------------------------------------------------------
+
+EOF
+}
+# Set default printed format
+table_format=true
+json_format=false
+csv_format=false
+
+# Parse command-line options
+REST_ARGS=()
+while [ $# -gt 0 ]; do
+  case "$1" in
+    -h|--help)
+      help
+      exit 0
+      ;;
+
+    -o|--output|-output)
+      if [ $# -gt 1 ]; then
+        if [ "$2" = "table" ]; then
+          table_format=true
+          json_format=false
+          csv_format=false
+        elif [ "$2" = "json" ]; then
+          table_format=false
+          json_format=true
+          csv_format=false
+        elif [ "$2" = "csv" ]; then
+          table_format=false
+          json_format=false
+          csv_format=true
+          csv_header=false
+        elif [ "$2" = "csvheader" ]; then
+          table_format=false
+          json_format=false
+          csv_format=true
+          csv_header=true
+        else
```

Comment:
- maybe it would improve readability if you set everything to false once (before line 42) and then set only the relevant variables to true in the if blocks?